src/BlockedUsers.tsx

```diff
@@ -0,0 +1,187 @@
+  const [blocked, setBlocked] = useState<{ [key: string]: any }>({});
+
+  const [loading, setLoading] = useState(false);
+  const [loadingMore, setLoadingMore] = useState(false);
+
+  const styles = setStyles(isDark);
+
+  const floatingMenu = useRef<IFloatingMenuRef>(null);
+  const flatList = useRef<any>(null);
+  const itemHeight = useRef(0);
+  const fListY = useRef(0);
+
+  const toggleBanEventListener = useCallback(
+    ({ user, type }: { user: any; type: string }) => {
+      const tempBlocked = { ...blocked };
+      if (type === 'user.banned') {
+        tempBlocked[user.id] = user;
+      }
+      if (type === 'user.unbanned') {
+        delete tempBlocked[user.id];
+      }
+      setBlocked(tempBlocked);
+    },
+    [blocked]
+  );
+
+  useEffect(() => {
+    client?.on(toggleBanEventListener);
+    client
+      .queryUsers({ banned: true }, {}, { limit: 100, offset: 0 })
+      .then(({ users }: { users: any[] }) => {
+        const tempBlocked = { ...blocked };
+        users.map((b: any) => (tempBlocked[b.id] = b));
+        setLoading(false);
+      });
+    return client?.off(toggleBanEventListener);
+  }, [blocked, client, toggleBanEventListener]);
+
+  const renderFLItem = useCallback(
+    ({ item }: { item: any }) => (
+      <ListItem
+        isDark={isDark}
+        appColor={appColor}
+        onLayout={({ nativeEvent: ne }: { nativeEvent: any }) =>
+          (itemHeight.current = ne.layout.height)
+        }
+        item={{ user: item }}
+        center
+        admin={admin}
+        type={'banned'}
+        onTap={() => floatingMenu.current?.close()}
+        onToggleBlockStudent={() => {
+          const tempBlocked = { ...blocked };
+          delete blocked[item.id];
+
+          setBlocked(tempBlocked);
+          onUnblockStudent(item);
+        }}
+      />
+    ),
+    [admin, appColor, blocked, isDark, onUnblockStudent]
+  );
+
+  const loadMore = useCallback(async () => {
+    setLoadingMore(true);
+
+    await client
+      .queryUsers({ banned: true }, {}, { limit: 100, offset: Object.keys(blocked).length })
+      .then(({ users }: { users: any }) => {
+        const tempBlocked = { ...blocked };
+        users.map((b: any) => (tempBlocked[b.id] = b));
+        setBlocked(tempBlocked);
+      })
+      .finally(() => setLoadingMore(false));
+  }, [blocked, client]);
+
+  return (
+    <>
+      <View style={{ flexDirection: 'row', alignItems: 'center' }}>
+        <TouchableOpacity
+          onPress={onBack}
+          style={{ padding: 10, flexDirection: 'row', alignItems: 'center' }}
+        >
+          {arrowLeft({ height: 12, width: 12, fill: '#4D5356' })}
+          <Text style={styles.titleText}>BLOCKED STUDENTS</Text>
+        </TouchableOpacity>
+      </View>
+      {loading ? (
+        <ActivityIndicator
+          size='large'
+          color={isDark ? 'white' : 'black'}
+          style={styles.activityIndicator}
+        />
+      ) : (
+        <>
```

Comment:
Is this extra needed?
`(
<>
....
</>
)
`

Reply:
Nope, removed it.